src/generators/Minecraft.res

```diff
@@ -144,6 +164,11 @@ module Cuboid = {
       right: Face.t,
       left: Face.t,
     }
+
+    type direction = [#East | #West | #North | #South]
```

Comment:
> Switched the placements of the back face for the "North" and "South" directions so that the specified direction indicates which direction the back face will be placed on, as before the "North" and "South" directions would place the back face on the opposite side to where it should be.

I'm curious about this change, any particular reason for this change? I'm guessing it might be this way because it's a simpler change tecnically?

The reason I ask is it would seem to make sense that the North, South, East, and West arrangement would be best applied to the "Center" face.

Reply:
Yeah, having the direction be the direction the back face was on was how I wrote it since that was easier when the other faces didn't move depending on the direction. It could be thought of either way, though, as long as the meanings of all directions are consistent with each other.